contributors/design-proposals/propagation.md

```diff
@@ -150,6 +165,42 @@ non-privileged', an environment check and an WARNING log will be emitted about
 whether propagation mode is supported.
 
 
+## Extra Concerns
+
+@lucab and @euank has some extra concerns about pod isolation when propagation
+modes are changed, listed below:
+
+1. how to clean such pod resources (as mounts are now crossing pod boundaries,
```

Comment:
Y, we already limited the usage to HostPath. I don't really understand why @euank is keen on the cleaning up issue.